### PR TITLE
bazel 5.3.2

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
still the linux issue, 

```
  Opening zip "/home/linuxbrew/.linuxbrew/Cellar/bazel/5.3.2/libexec/bin/bazel-real": Cannot find central directory
  FATAL: Failed to open '/home/linuxbrew/.linuxbrew/Cellar/bazel/5.3.2/libexec/bin/bazel-real' as a zip file: (error: 5): Input/output error
```

File: Formula/bazel.rb [hunk 1]
```diff
@@ -89,18 +89,5 @@ def install
...
-    # bypasses this behavior.
-    (testpath/"tools"/"bazel").write <<~EOS
-      #!/bin/bash
-      echo "stub-wrapper"
-      exit 1
-    EOS
-    (testpath/"tools/bazel").chmod 0755
-
-    assert_equal "stub-wrapper\n", shell_output("#{bin}/bazel --version", 1)
-    assert_equal "bazel #{version}-homebrew\n", shell_output("#{bin}/bazel-#{version} --version")
```
Comment (carlocab) [inline Formula/bazel.rb:104]: I don't think we want to remove this test entirely. Just `return if OS.linux?` instead.

Comment (chenrui333) [reply Formula/bazel.rb:104]: yeah, let's see how it goes, and I can add `return if OS.linux? ` before this block.

**Comment by chenrui333** [comment]
ok, still failed. I will just do the return in the beginning then.

**Pull request closed without merging**